### PR TITLE
Fix config value for tikv compression-per-level

### LINKED ISSUE #7842: Inconsistent description of the default value of TiKV's config `rocksdb.*.compression-pre-level`
## Error 报告

**This repository is ONLY used to solve problems related to DOCS-CN.
For other issues (TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. What is the document directory for the problem?

https://docs.pingcap.com/zh/tidb/v5.2/tikv-configuration-file#compression-per-level

image

2. How would you like to improve it?

Make the default value correct.

As I see in master branch of TiKV, the default config is `["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]` for write_cf and default_cf and all `"no"`s for lock_cf.

### PR DESCRIPTION
**Pull request by tabokie**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Fix #7842

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
  - But no need to translate to v4.0, as the value is correct for docs-v4.0
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

No

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1040,7 +1040,9 @@ bloom filter 为每个 key 预留的长度。
 ### `compression-per-level`
 
 + 每一层默认压缩算法，默认：前两层为 No，后面 5 层为 lz4。
```
Comment (MyonKeminta) [inline tikv-configuration-file.md:1042]:
```suggestion
+ 每一层默认压缩算法。
```

**Review by MyonKeminta — COMMENTED** [review]
Rest LGTM

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1039,8 +1039,10 @@ bloom filter 为每个 key 预留的长度。
 
 ### `compression-per-level`
 
-+ 每一层默认压缩算法，默认：前两层为 No，后面 5 层为 lz4。
-+ 默认值：["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
++ 每一层默认压缩算法。
++ `defaultcf` 默认值：["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
++ `writecf` 默认值：["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
++ `lockcf` 默认值：["no", "no", "no", "no", "no", "no", "no"]
```
Comment (TomShawn) [inline tikv-configuration-file.md:1045]:
```suggestion
+ `defaultcf` 的默认值：["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
+ `writecf` 的默认值：["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
+ `lockcf` 的默认值：["no", "no", "no", "no", "no", "no", "no"]
```

**Review by TomShawn — APPROVED** [review]

**Review by ngaut — APPROVED** [review]

**Pull request merged**